### ISSUE TITLE
Appimage does not draw modules in timeline

### ISSUE THREAD

**Issue by JRSV** [ossia/score#1051]
I have downloaded score latest V3 alpha, as an appimage in fedora 31, unfortunately when I add modules to the time line, they are not drawn. If I place a sound genrator and press play I can hear it is working.

any ideas what is going on?

**Comment by jcelerier** [comment]
Hello, can you try with the latest appimage ? thanks !

**Comment by jcelerier** [comment]
Also, can you post a quick glxinfo log ? The software uses opengl for the timeline now, it may be related... 
Are you running under Wayland or X11 ?

**Comment by JRSV** [comment]
Ok, this version seems to work, although, it does not allow me to rezise the main window vertically, its a bit weird, the maximize button works on my second display but when I press it in my first display it disappears.. I am running X11 , do you need my glxinfo log still? (its big)

**Comment by jcelerier** [comment]
> its a bit weird, 

that's putting it lightly, from the sound of it that feels like a Qt bug.
May I ask for a video of the behaviour ? (I generally use simplescreenrecorder if you don't have a screencap program).

That doesn't sound like a GL bug in any case so glad that one has been fixed.

**Comment by JRSV** [comment]
Sure, how should I share a video with you though??

**Comment by jcelerier** [comment]
if there's nothing personal on screen you can put it on https://streamable.com/ - else send it to me through wetransfer maybe ? jeanmichael.celerier@gmail.com

**Comment by JRSV** [comment]
Video of Screen1, where the maximize button disappears.
https://streamable.com/ac4b3h
Video of screen2, where the maximize button appears.
https://streamable.com/neof3d

**Comment by jcelerier** [comment]
hum, I'm really at loss with that one :( windows chrome (where there are the minimize / maximize buttons) is not something that is handled by score itself but by the compositor... I've looked a bit around here for issues but did not find anything though : https://www.google.com/search?q=maximize+button+disappears+fedora+dual+screen

**Comment by JRSV** [comment]
oh well... hopefully there will be no more surprises. I don't think it actually affects the program itself.
thanks

**Comment by jcelerier** [comment]
does this still happen with the Qt 6 version ?

**Comment by JRSV** [comment]
now I can resize the window, this app is looking better and better, any intentions to implement video in pipewire? anyway, thanks for this.
cheers

**Comment by jcelerier** [comment]
yes ! pipewire video is in the pipeline, there is already pipewire audio and pipewire midi so hopefully video won't be too hard to add !
